float32 a1
float32 a2
float32 b1
float32 b2
float32 ykm1
float32 ykm2
float32 ukm1
float32 ukm2
---
float32 yk
